app/scripts/views/mailer/resetPassword.ejs

```diff
@@ -0,0 +1,18 @@
+Pepyatka password reset
+------------------------
+
+Thank you for using Pepyatka. We received a password reset request
+for this email address (<%= user.email %>) on
+https://try.pepyatka.com/. To reset your password, click the link below:
+
+https://try.pepyatka.com/account/reset?token=<%= user.resetPasswordToken %>
+
+If you did not request a password reset, then someone probably
+mis-typed their email address. You can ignore this message, and we
+apologize for the inconvenience.
+
+If you have any problems resetting your password, you can get help
+from us at mail@pepyatka.com.
```

Comment:
* `try.pepyatka.com` in URLs (2 times)
* `mail@pepyatka.com` email (1 time)

both should be configurable